registry/service.go

```diff
@@ -126,12 +126,7 @@ func (s *DefaultService) Auth(ctx context.Context, authConfig *types.AuthConfig,
 	}
 
 	for _, endpoint := range endpoints {
-		login := loginV2
-		if endpoint.Version == APIVersion1 {
-			login = loginV1
```

Comment:
Wondering if V1 login should still be a thing (IIRC "authentication" is separate from the registry specs, but I may be wrong here)